tests/test_cases/split/simple_size2.bs

```diff
@@ -0,0 +1,13 @@
+manifest substance_a
+manifest dilutant1
+module fluorescence
+
+instructions:
+
+sa = dispense 10 units of substance_a
+d1 = dispense 1 units of dilutant1
+
+first_dilute = mix sa with d1 for 5s
+x = split first_dilute into 2
+dispose x[0]
```

Comment:
If you are disposing (or performing an identical operation) on all droplets resulting from a split, you should us a SIMD instruction:
example: 
x = split first_dilute into 2
dispose x // this instruction disposes both droplets.